gen/golden_test.go

```diff
@@ -0,0 +1,67 @@
+package gen
+
+import (
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func hash(t *testing.T, name string) string {
+	f, err := os.Open(name)
+	defer f.Close()
```

Comment:
put `defer` after the `require.NoError` (otherwise it'll panic if the file isn't found)